src/pages/ResourceBatchUpdatePage.tsx

```diff
@@ -335,7 +368,76 @@ const ResourceBatchUpdatePage = ({
       <section className="section-spans">
         <h2>Joukkopäivitykseen valitut aukiolot</h2>
         <p>Olet valinnut joukkopäivitykseen alla olevat aukioloajat.</p>
-        {/* this block will be implemented later */}
+
+        {normalSelectedDatePeriods.length > 0 && (
+          <OpeningPeriodsSection
+            datePeriods={normalSelectedDatePeriods}
+            isLoading={false}
+            newUrl=""
+            theme="DEFAULT"
+            title="Aukioloajat"
+            addDatePeriodButtonText=""
+            addNewOpeningPeriodButtonDataTest="">
+            {normalSelectedDatePeriods.map((dp) => (
+              <OpeningPeriod
+                key={dp.id}
+                datePeriod={dp}
+                language={language}
+                initiallyOpen={false}
+                editUrl=""
+                deletePeriod={() => {
+                  return Promise.resolve();
```

Comment:
Doesn't seem to be doing anything, is this perhaps still to be implemented later? 🤔